README.md

```diff
@@ -290,6 +293,7 @@ assert(Object.is(#[NaN], #[NaN]));
 // SameValueZero
```

Comment:
I think we can get rid of this comment:
```suggestion
```

Reply:
I guess the comment exists to indicate that the SameValueZero algorithm is used by Maps (since that's not well-known to all JS programmers).

Reply:
Ok so maybe we could factor [this list](https://github.com/tc39/proposal-record-tuple/pull/130#discussion_r435213074) inside of the examples as comments, that should dissipate all confusions and I would visually like that better in the proposal

Reply:
```suggestion
// SameValueZero is still SameValueZero and is performed recursively
```